CMakeLists.txt

```diff
@@ -184,6 +184,15 @@ if(UNIX)
        include(Coveralls)
        coveralls_turn_on_coverage()
     endif()
+else(UNIX)
+    if(NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+        message(FATAL_ERROR "RINGMesh on Windows requires Microsoft Visual C++")
```

Comment:
I think it is possible to compile under Windows without Visual. Should we forbid to use something else?

Reply:
Indeed it is possible. But for now it is not supported. The day it is supported to use something else, we will see.

Reply:
Ok